report_complete_assignments_without_work_orders.md

```diff
@@ -0,0 +1,43 @@
+## Report Complete Assignments without Completed Work Orders
+
+This script takes assignments that have been completed (status = "completed") that do not have a corresponding completed work order (Survey or Collector feature) and reports them to the user. 
+
+A work order is defined as a feature (row in the attribute table) that corresponds to work integrated with a Workforce assignment. For example, an inspection survey may need to be completed by a field worker at each Workforce assignment.
+
+Whether or not the user has a corresponding work order is defined by searching a particular field in the survey/collector feature layer for the value stored in the assignment's work order id.
+
+For example, if I have an assignment with the work_order_id "6" that has status "completed", but there is not a survey that has already been submitted with the value 6 in the field I'm using to integrate the two apps, then this is an assignment that will get logged. You provide the name of that field the script searches on.
+
+Note that an assignment will only get reported if it has a value for Work Order ID.
+
+This script assumes a 1:1 relationship between surveys/features and assignments.
+
+**This script requires the logged in user to be an admin or to be the owner of the project**
```

Comment:
Why?  I'm not seeing anything in here that should require that level of privileges.  Looks like you just need the ability to query a couple items and layers.